### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by grace-mugoiri**
Great day to work with forms.

### CODE REVIEW

**Comment by nidalaa** [comment]
Please follow the instructions about stickler configuration from here https://github.com/microverseinc/linters-config/tree/master/css.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title></title>
```
Comment (JamesOkunlade) [inline index.html]: The title should not be left empty.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        <div class="container">
+            
+            <h4>One Account.</h4>
+            <h5> Everything Intuit.</h5>
+            <p>Create and account for all intuits's products</p> 
+            <p>including Mint.<a href="#" > Learn More</a></p>
+
+            <form> 
+                <label>Email address </label><br>
+                <input>
```
Comment (JamesOkunlade) [inline index.html]: Specify type and name attributes for all input elements.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+                <br>
+                <label id="phone-label">Phone (Recommended) </label><br>
+                <input>
+                <br>
+                <label for="Password">Password</label><br>
+                <input type="password" name="password" id="password"><br>
+                <input type="submit" value="Create Account" id="submit">
+
+                <p> <i> By clicking Create Account, you agree to our</p>
```
Comment (JamesOkunlade) [inline index.html]: The open tag "i" does not have a corresponding closing tag.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+                <p><a href="#">Terms of service</a> and have read and ackowledge</p>
+                <p>our <a href="#">Privacy Statement</a></i></p>
```
Comment (JamesOkunlade) [inline index.html]: Element "p" not allowed as child of the element "i" in this context. (Suppressing further errors from this subtree.)
It violates nesting rules.

**Review by JamesOkunlade — CHANGES_REQUESTED** [review]
Hello guys! Good work on the html-forms project. However, there are some changes you need to make as described in the comments.

You should always run your code through the W3C markup validator before requesting for code review to ensure your markups are without errors.

Effect the changes and send me another code review request.

**Comment by grace-mugoiri** [comment]
we have made the requested changes.

**Review by JamesOkunlade — CHANGES_REQUESTED** [review]
I viewed your form in a browser and realised it doesn't look much like the mint sign up form.

1. The logo three icons on the form should be grabbed from the mint sign up page and added to your form.

2. Also, the sign in button and text at the top right corner of the page should be created in the project as well.

Please correct these changes and submit for another code review. 

Happy coding! :sunglasses:

**Comment by grace-mugoiri** [comment]
-Fixed the form from scratch using CSS and HTML only. 
-Stickler still wont pass the review.

**Review by JamesOkunlade — APPROVED** [review]
### Looks perfect!! :sunglasses:
Merge the develop to master, close the PR and submit completion form. 

Regarding the stickler setup, there should be someone in your standup that has done it right. Ask for help from your team or bring it up in the full-time-students slack channel.

Good luck in your future projects. :+1::+1:

**Pull request merged**